app/controllers/evaluation_forms_controller.rb

```diff
@@ -4,6 +4,9 @@
 class EvaluationFormsController < ApplicationController
   helper FormHelper
 
+  include EvaluationFormsHelper
+  include PhasesHelper
```

Comment:
it's preferable to use the `helpers` object instead of including the helper functions into the controller

Reply:
```suggestion
```